### PR TITLE
Add ubuntu-sdk-16.04

### PR DESCRIPTION
**Pull request by mariogrip**

### CODE REVIEW

File: clickreviews/cr_security.py [hunk 1]
```diff
@@ -132,6 +132,9 @@ def __init__(self, fn, overrides=None):
             'ubuntu-sdk-15.04': {
                 'policy_version': 1.3,
             },
+            'ubuntu-sdk-16.04': {
+                'policy_version': 16.04,
```
Comment (NeoTheThird) [inline clickreviews/cr_security.py:136]: shouldn't it be 1.0?

Comment (dpniel) [reply clickreviews/cr_security.py:136]: @NeoTheThird 16.04 is fine as it references this template https://github.com/ubports/apparmor-easyprof-ubuntu/tree/xenial/data/templates/ubuntu/16.04

Comment (dpniel) [reply clickreviews/cr_security.py:136]: LGTM :+1:

**Review by dpniel — APPROVED** [review]

**Review by sverzegnassi — APPROVED** [review]
LGTM.

**Pull request merged**